test/powershell/engine/Module/XamlModuleNotSupported.Tests.ps1

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+Describe "Workflow .Xaml module is not supported in PSCore" -tags "CI" {
```

Comment:
Why not have this section in the existing `Import-Modules.Tests.ps1` instead of a new script?

Reply:
Good point. Will move the tests there.